### DISCUSSION TITLE
Whether this dataset can be used for non-commercial purposes

### DISCUSSION THREAD

**Discussion by Lunahannen** [PaddlePaddle/PaddleSpatial#18] [BaiduKDDCup2022] [answered]
Dear organizers,
  Whether this dataset can be used for non-commercial purposes.For example, whether this dataset can be used in a dissertation.
  thanks.

**Comment by zhoujingbo** [comment, marked as answer]
Hi, 

The dataset can be used for non-commercial purposes. If you use this dataset, please cite our report:

> Jingbo Zhou, Xinjiang Lu, Yixiong Xiao, Jiantao Su, Junfu Lyu, Yanjun Ma, Dejing Dou; SDWPF: A Dataset for Spatial Dynamic Wind Power Forecasting;  Technical Report, 2022. 

According to your concern, we will upload our report to arxiv later for citation purposes.

Thanks a lot.
Regards
Jingbo
